### DISCUSSION TITLE
Major differences with V8?

### DISCUSSION THREAD

**Discussion by automatedmachine** [microsoft/fluentui#16336] [General] [answered]
I'm starting a new project using fluentui/react and I'm curious about v8 beta. Is there significant changes or new features in the new version? Also, what's the projected timeline to release as stable?

It might be worthwhile to begin developing using v8 beta right now vs using v7 and upgrading in the future, but of course stability is a concern in a beta.

Does anyone have any info on the next version? I looked at the project page [(https://github.com/microsoft/fluentui/projects/36)](**url**) but it didn't really answer these questions.

**Comment by JustSlone** [comment, marked as answer]
@automatedmachine take a look at the pinned issue: #12770 that, plus the release notes/migration guide should cover both what's changed as well as the major differences between the two. 

There aren't a ton of differences, but also the v8 beta is pretty stable and we'd ♥ any feedback you have on it!
